vizro-core/src/vizro/models/_components/form/range_slider.py

```diff
@@ -136,7 +136,11 @@ def build(self):
                                     if self.step
                                     else "slider_input_field_no_space_right",
                                 ),
-                                dcc.Store(id=f"temp-store-range_slider-{self.id}", storage_type="session"),
+                                dcc.Store(
+                                    id=f"{self.id}_input_store",
+                                    storage_type="session",
+                                    data=self.value or [value[0], value[1]],
+                                ),
```

Comment:
At the beginning of the `build` method, a variable called `value` is calculated. So you can probably just reuse it and set it like this:
```
                                dcc.Store(
                                    id=f"{self.id}_input_store",
                                    storage_type="session",
                                    data=value,
                                ),
```

Reply:
We can also probably avoid ambiguity by renaming the `value` variable to something like `init_value` or whatever makes more sense to you.